airflow/www/decorators.py

```diff
@@ -88,8 +88,10 @@ def wrapper(*args, **kwargs):
             with create_session() as session:
                 if not get_auth_manager().is_logged_in():
                     user = "anonymous"
+                    user_display = None
```

Comment:
I wonder if it’s a good idea to introduce None (a different type) here; either an empty string or `"Anonymous"` would be easier for consuming processes to handle.

Reply:
Agreed, good catch. `get_user_display_name` returns an empty string when empty so this should do the same.